ui/src/test/java/org/pentaho/mantle/client/dialogs/scheduling/ScheduleEditorTest.java

```diff
@@ -452,7 +452,8 @@ public void testGetStartDate() throws Exception {
     Calendar calendarRequest = Calendar.getInstance();
     calendarRequest.setTime( date );
     when( scheduleEditor.runOnceEditor.getStartDate() ).thenReturn( calendarRequest.getTime() );
-    when( scheduleEditor.runOnceEditor.getStartTime() ).thenReturn( "2:50:33 " + TimeUtil.TimeOfDay.PM.toString() );
+    String startTime = "2:50:33 " + TimeUtil.TimeOfDay.PM.toString();
+    when( scheduleEditor.runOnceEditor.getStartTime() ).thenReturn( startTime );
```

Comment:
Mockito complains when you put an enum directly inside a thenReturn; needs to be assigned to a variable first.